lab_7_llm/main.py

```diff
@@ -4,7 +4,7 @@
 # pylint: disable=too-few-public-methods, undefined-variable, too-many-arguments, super-init-not-called
```

Comment:
But why are these changes from lab  7 still here?

Reply:
It should be ok now